math_.py

```diff
@@ -13,3 +13,14 @@ def substract(a: int, b: int) -> int:
 def list_substract(a: List, b: List) -> List:
     assert len(a)==len(b), "len(a) != len(b)"
     return [i-j for i, j in zip(a,b)]
+
```

Comment:
The function name should be more descriptive, e.g., `is_numeric` or `is_integer_or_string_convertible_to_integer`.